pkg/buildcontext/buildcontext.go

```diff
@@ -51,6 +51,8 @@ func GetBuildContext(srcContext string) (BuildContext, error) {
 			return &AzureBlob{context: srcContext}, nil
 		}
 		return nil, errors.New("url provided for https context is not in a supported format, please use the https url for Azure Blob Storage")
+	case constants.FileBuildContextPrefix:
```

Comment:
Can we rename this to `tar` instead of `file` to be more sepcific that the only file format supported is `tar.gz`